src/Components/test/E2ETestMigration/Microsoft.AspNetCore.Components.Migration.E2ETests.csproj

```diff
@@ -9,9 +9,6 @@
     <!-- This project references the shared framework transitively. Prevent restore errors by setting this flag. -->
     <GenerateErrorForMissingTargetingPacks>false</GenerateErrorForMissingTargetingPacks>
 
-    <TestTrimmedApps Condition="'$(ContinuousIntegrationBuild)' == 'true'">true</TestTrimmedApps>
-    <TestTrimmedApps Condition="'$(Configuration)' == 'Release'">true</TestTrimmedApps>
```

Comment:
Does this mean we never test trimmed apps anymore?

Reply:
This is disabling it specifically for the `Migration` E2E tests (the ones written in playwright). These tests are a duplicate of our Selenium based tests, which still test the trimmed apps in the `components-e2e` pipeline:

https://github.com/dotnet/aspnetcore/blob/f8631573c0cecc2d4f732673af600aea3968935c/src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj#L29-L30

Reply:
Thanks!